hack/generate-template-variable-doc.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
```

Comment:
4 spaces indentation for python code

https://peps.python.org/pep-0008/#indentation